services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers_long_running_tasks.py

```diff
@@ -332,4 +292,4 @@ async def containers_restart_task(
         )
         return task_id
     except TaskAlreadyRunningError as e:
-        raise HTTPException(status.HTTP_409_CONFLICT, detail=f"{e}") from e
+        return e.managed_task.task_id  # pylint: disable=no-member
```

Comment:
so all the routes on the dynamic sidecar now are **not** raising a conflict when the client calls several time the same function? Nevertheless it would be good to be able to trace if the director-v2 takes the bad habit of calling several times for nothing.. this wastes resources on both director-v2 client and sidecar server connections...